src/controllers/user/Controller.ts

```diff
@@ -73,6 +76,47 @@ class UserController {
       }
     }
 
+    login(req: Request , res: Response , next: NextFunction) {
+      try { const { email, password } = req.body;
+      userModel.findOne ( { email }, (err, result) => {
+          if ( result ) {
+              if ( password === result.password ) {
+                  console.log('result is', result.password);
+                  const token = jwt.sign({
+                      ...result.toObject()
+                  }, config.SECRET_KEY);
+                  console.log( token );
+                  res. send( {
+                      data: { ...result.toObject(), token },
+                      message: 'Login Permitted',
+                      status: 200
+                  });
+              }
+              else {
+                  res.send ( {
+                      message: 'id doesnt match',
+                      status: 400
+                  });
+              }
+          } else {
+              res.send ( {
+                  message: ' Email is not registered ',
+                  status: 404
+              });
+          }
+      });
+      }
+      catch ( err ) {
+          res.send( err );
+      }
+
+    }
+    me( req: Request, res: Response, next: NextFunction) {
+        res.json( {
+            // data
+        });
```

Comment:
Implement this route to return the login user details based on the token provided in the request.